Beeper/barcelona-mautrix/BarcelonaMautrix.swift

```diff
@@ -57,8 +51,37 @@ class BarcelonaMautrix {
         } else {
             CLInfo("CoreSentry", "Starting without setting up Sentry")
         }
+    }
+    
+    static func getUnixSocketPath() -> Optional<String> {
+        let index = ProcessInfo.processInfo.arguments.firstIndex(of: "--unix-socket")
+        if let index {
+            return ProcessInfo.processInfo.arguments[index + 1]
```

Comment:
This will cause a crash on the rare instance that someone passes in `--unix-socket` without passing in a path after it - should we instead check `if let index, ProcessInfo.processInfo.arguments.count > index {`?

Reply:
Always better to check and give a clear error, I think. CLI inputs are easy to get wrong.